.circleci/config.yml

```diff
@@ -51,6 +51,11 @@ jobs: # a collection of steps
           name: Database setup
           command: bin/rake db:schema:load --trace
 
+      - run:
+          name: Run latest migrations
+          command: |
+            STEP=2 bundle exec rake db:rollback db:migrate
```

Comment:
Curious why we rollback, then migrate here?

Reply:
It's because we check in the latest `db/schema.rb` (which is normal in general, but not always at AppFolio).
So the schema is already up-to-date without running any migrations.